fabfile.py

```diff
@@ -228,6 +229,33 @@ def upgrade_binary(conn):
     conn.run(f'dtach -n `mktemp -u /tmp/dtach.XXXX` sh /home/ubuntu/cmd.sh')
 
 
+# ======================================================================================
+#                                       flooder
+# ======================================================================================
+
+
+@task
+def send_flooder_binary(conn):
+    # 1. send new binary
+    zip_file = 'flooder.zip'
+    cmd = f'zip -j {zip_file} bin/flooder'
+    call(cmd.split())
+    conn.put(f'{zip_file}', '.')
+    conn.run(f'unzip -o /home/ubuntu/{zip_file} && rm {zip_file}')
```

Comment:
since this code is almost identical to `send_new_binary`, it could be extracted to a common method

Reply:
done